Submitted By:     Nagy Bence <nagybence@tipogral.hu>
Date:             2005.02.03
Initial Packages: 2.9.5
Upstream Status:  Not submitted - Frugalware-specific
Description:      This patch adds a new feature to makepkg.
                  If available, use Frugalware's extensinons to makepkg.
                  (provided by the pacman-tools package)
Update:           Also defines a fake Finclude function to disable inclusion of
                  these files when generating/updating a .fdb.

diff -Naur pacman-2.9.6.orig/scripts/gensync pacman-2.9.6/scripts/gensync
--- pacman-2.9.6.orig/scripts/gensync	2005-06-10 22:54:29.000000000 +0200
+++ pacman-2.9.6/scripts/gensync	2005-07-25 09:58:17.000000000 +0200
@@ -51,6 +51,10 @@
 	exit 1
 }
 
+error() { return 0; }
+Fdie() { return 0; }
+Finclude() { return 0; }
+
 get_md5checksum()
 {
 	if [ "$pkgdir" != "" ]; then
diff -Naur pacman-2.9.6.orig/scripts/makepkg pacman-2.9.6/scripts/makepkg
--- pacman-2.9.6.orig/scripts/makepkg	2005-06-10 22:54:29.000000000 +0200
+++ pacman-2.9.6/scripts/makepkg	2005-07-25 09:56:42.000000000 +0200
@@ -25,6 +25,9 @@
 PKGDEST=$startdir
 USE_COLOR="n"
 
+# Frugalware extensions to makepkg
+[ -e /usr/lib/frugalware/fwmakepkg ] && . /usr/lib/frugalware/fwmakepkg
+
 # source Arch's abs.conf if it's present
 [ -f /etc/abs/abs.conf ] && source /etc/abs/abs.conf
 
@@ -301,7 +304,7 @@
 fi
 
 unset pkgname pkgver pkgrel pkgdesc url license groups provides md5sums force
-unset replaces depends conflicts backup source install build makedepends
+unset replaces depends conflicts backup source install makedepends
 umask 0022
 
 if [ ! -f $BUILDSCRIPT ]; then
diff -Naur pacman-2.9.6.orig/scripts/updatesync pacman-2.9.6/scripts/updatesync
--- pacman-2.9.6.orig/scripts/updatesync	2005-06-10 22:54:29.000000000 +0200
+++ pacman-2.9.6/scripts/updatesync	2005-07-25 09:58:29.000000000 +0200
@@ -54,6 +54,8 @@
 	exit 1
 }
 
+Finclude() { return 0; }
+
 get_md5checksum()
 {
 	if [ "$pkgdir" != "" ]; then
